src/Minigames/Number_Guesser.py

```diff
@@ -0,0 +1,41 @@
+class Number_Guesser:
+    def __init__(self):
+        self.target_number = None
+        self.max_tries = 3
+        self.current_attempts = 0
+        self.ready_players = None
+        self.players = []
```

Comment:
There is no need to keep track of player uuids here. This should be done in the UI class.